go.sum

```diff
@@ -0,0 +1,41 @@
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
These changes should also be separate.

Reply:
Ok

Reply:
:+1:

Reply:
Isn't it dangerous to not have a go.sum ?

Reply:
I don’t know anything about go modules. Back when I wrote this, the Go team hadn’t yet conceded that versioning was a useful concept.

In any case, these changes don’t belong in this PR.